product_pricelist_supplierinfo/models/product_product.py

```diff
@@ -24,3 +24,33 @@ def price_compute(self, price_type, uom=False, currency=False,
             return dict.fromkeys(self.ids, 1.0)
         return super().price_compute(
             price_type, uom=uom, currency=currency, company=company)
+
+    def _select_seller(self, partner_id=False, quantity=0.0, date=None, uom_id=False,
```

Comment:
Params? Better using the actual param name (`avoid_min_qty`), this is too generic

Reply:
I'm using the [original parameters](https://github.com/odoo/odoo/blob/f23e07dcb0606dbc4bf835b8266323d5a515662d/addons/product/models/product.py#L566), and `params` is one of them, I'm only using it as is used on the [`purchase` module](https://github.com/odoo/odoo/blob/f23e07dcb0606dbc4bf835b8266323d5a515662d/addons/purchase/models/purchase.py#L632)

Reply:
Oh, I see. Kind of weird that original method, but it's OK for me then.